### ISSUE TITLE
ctffind 4.1.5 returns non-zero exit status 255

### ISSUE THREAD

**Issue by azazellochg** [azazellochg/scipion#146]
Only for ctffind 4.1.5. 
It finishes correctly, but check_call(command, shell=True, stdout=sys.stdout, stderr=sys.stderr, env=env, cwd=cwd) returns this error. No error when running outside scipion. Also no error with ctffind 4.0.15

Traceback:
```
00098:   Traceback (most recent call last):
00099:     File "/home/sharov/soft/scipion/pyworkflow/protocol/protocol.py", line 182, in run
00100:       self._run()
00101:     File "/home/sharov/soft/scipion/pyworkflow/protocol/protocol.py", line 228, in _run
00102:       resultFiles = self._runFunc()
00103:     File "/home/sharov/soft/scipion/pyworkflow/protocol/protocol.py", line 224, in _runFunc
00104:       return self._func(*self._args)
00105:     File "/home/sharov/soft/scipion/pyworkflow/em/packages/grigoriefflab/protocol_ctffind.py", line 120, in _estimateCTF
00106:       self.runJob(self._program, self._args % self._params)
00107:     File "/home/sharov/soft/scipion/pyworkflow/protocol/protocol.py", line 1015, in runJob
00108:       self._stepsExecutor.runJob(self._log, program, arguments, **kwargs)
00109:     File "/home/sharov/soft/scipion/pyworkflow/protocol/executor.py", line 56, in runJob
00110:       env=env, cwd=cwd)
00111:     File "/home/sharov/soft/scipion/pyworkflow/utils/process.py", line 51, in runJob
00112:       return runCommand(command, env, cwd)
00113:     File "/home/sharov/soft/scipion/pyworkflow/utils/process.py", line 65, in runCommand
00114:       check_call(command, shell=True, stdout=sys.stdout, stderr=sys.stderr, env=env, cwd=cwd)
00115:     File "/home/sharov/soft/scipion/software/lib/python2.7/subprocess.py", line 540, in check_call
00116:       raise CalledProcessError(retcode, cmd)
00117:   CalledProcessError: Command 'export OMP_NUM_THREADS=1; /home/sharov/soft/scipion/software/em/ctffind4/ctffind  << eof
...
00138:   ' returned non-zero exit status 255
```

**Comment by azazellochg** [comment]
**Update:** this is a ctffind4 bug. Present when running outside scipion.

**Comment by azazellochg** [comment]
Will be fixed in ctffind 4.1.6, coming soon
